src/framework/preferences/js/StarterGrades.js

```diff
@@ -97,23 +97,15 @@ var fluid_2_0 = fluid_2_0 || {};
 
     fluid.defaults("fluid.uiEnhancer.starterEnactors", {
         gradeNames: ["fluid.uiEnhancer", "fluid.uiEnhancer.cssClassEnhancerBase", "fluid.uiEnhancer.browserTextEnhancerBase", "autoInit"],
-        connectionsGrade: "fluid.prefs.uiEnhancerConnections",
-        distributeOptions: {
-            source: "{that}.options.connectionsGrade",
-            removeSource: true,
-            target: "{that > fluid.prefs.enactor}.options.gradeNames"
-        },
+        model: "{fluid.prefs.rootModel}.rootModel",
         components: {
             textSize: {
                 type: "fluid.prefs.enactor.textSize",
                 container: "{uiEnhancer}.container",
                 options: {
                     fontSizeMap: "{uiEnhancer}.options.fontSizeMap",
-                    rules: {
-                        "textSize": "value"
-                    },
                     model: {
-                        value: "{fluid.prefs.rootModel}.rootModel.textSize"
+                        value: "{uiEnhancer}.model.textSize"
```

Comment:
This is odd - why does the parent's component model bind to {fluid.prefs.rootModel} whereas all the subcomponents bind to {uiEnhancer}.model?